docs/exploring-es-data.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[exploring-es-data]]
+= Exploring data in Elasticsearch
+
+[partintro]
+--
+By default Elastic APM data are stored in different index types. 
```

Comment:
different index types -> different indices

Reply:
Should be "data is stored"